Added travis, migrate to PW 1.4.2, change runner in components tests (#411)
V4Fire/Client: src/base/b-virtual-scroll/CHANGELOG.md

> - :house:      [Internal]
> - :nail_care:  [Polish]

> 

Reviewer: ```suggestion
```

## v3.0.0-rc.81 (2020-10-08)

#### :bug: Bug Fix

> * Fixed an issue with `renderNext` was not rendering data after loading error

Reviewer: ```suggestion
* Fixed an issue with `renderNext`: hasn't been data rendering after a loading error
```
